src/test/java/org/apache/commons/crypto/stream/AbstractCipherStreamTest.java

```diff
@@ -282,6 +570,24 @@ protected CryptoInputStream getCryptoInputStream(final ByteArrayInputStream bais
         return new CryptoInputStream(bais, cipher, bufferSize,
                 new SecretKeySpec(key, "AES"), new IvParameterSpec(iv));
     }
+    
+    protected CryptoInputStream getCryptoInputStream(final String transformation, final Properties props, 
+    	    final ByteArrayInputStream bais, final byte[] key, final AlgorithmParameterSpec params,
+    	    boolean withChannel) throws IOException {
+        if(withChannel) {
```

Comment:
```suggestion
        if (withChannel) {
```

But can't you just call the other `getCryptoInputStream`?